Loop/Settings/MoreSettingsView.swift

```diff
@@ -59,13 +59,13 @@ struct MoreSettingsView: View {
             })
 
             Section("Stage Manager") {
-                Toggle("Respect Stage Manager", isOn: $respectStageManager)
+                Toggle("Respect stage manager", isOn: $respectStageManager)
```

Comment:
"stage manager" should be title-cased to "Stage Manager"

Reply:
Updated.